sstables/storage.cc

```diff
@@ -133,7 +133,12 @@ void filesystem_storage::open(sstable& sst) {
     // sstable being created in parallel with the same generation.
     file_output_stream_options options;
     options.buffer_size = 4096;
-    auto w = sst.make_component_file_writer(component_type::TemporaryTOC, std::move(options)).get0();
```

Comment:
In the patch comment you say

> when filesystem_storage calls into sstable, it uses the generic backend to create the temporary toc instead of the underlying filesystem backend

I'm not sure I follow this. When the fs storage calls into sstable it uses the _sstable's_ _backend_, not the _generic_ one (btw -- what is the "generic" backend?). Please, explain

Reply:
sure. the word "generic" is way too generic, and not very specific. i revised the commit message. could you take another look?